app/frontend/src/utils/timeAgo.ts

```diff
@@ -12,13 +12,20 @@ export const twoMonthMillis = weekMillis * 8;
 export const yearMillis = 31557600000;
 export const twoYearMillis = 31557600000 * 2;
 
-export function timeAgo(input: Date | string, fuzzy: boolean = false) {
+export const lessThanHour = "Less than an hour ago";
+
+export function timeAgo(
+  input: Date | string,
+  fuzzyMillis?: number,
+  fuzzyText?: string
```

Comment:
I think it's safer to have this as one optional `fuzzy` parameter, an object with required `millis` and `text`, so it's impossible to accidentally set only one.